docs/index.md

```diff
@@ -12,6 +12,7 @@ Below, you'll find the following sections:
   - [Chaincode developer guide](#chaincode-developer-guide)
   - [API developer guide](#api-developer-guide)
 - [Operations guide](#operations-guide)
+- [Continuous Integration](https://github.com/hyperledger/fabric/blob/master/TravisCI_Readme.md)
```

Comment:
@christo4ferris : I thought this was missed in previous fixes. Now, I completely agree with you.  Will revert the changes related to Travis doc.